build.sbt

```diff
@@ -82,22 +85,29 @@ lazy val microservice = Project(appName, file("."))
   .enablePlugins(play.sbt.PlayScala, SbtDistributablesPlugin)
   .disablePlugins(JUnitXmlReportPlugin)
   .settings(PlayKeys.playDefaultPort := 9165)
-  .settings(coverageSettings: _*)
-  .settings(playSettings: _*)
+  .settings(coverageSettings *)
+  .settings(playSettings *)
   .settings(majorVersion := 1)
-  .settings(scalaSettings: _*)
-  .settings(defaultSettings(): _*)
+  .settings(scalaSettings *)
+  .settings(defaultSettings() *)
   .settings(
     Test / Keys.fork := true,
-    scalaVersion := "2.13.12",
+    scalaVersion     := "2.13.16",
     libraryDependencies ++= appDependencies,
     retrieveManaged := true
   )
   .configs(IntegrationTest)
-  .settings(inConfig(IntegrationTest)(Defaults.itSettings): _*)
+  .settings(inConfig(IntegrationTest)(Defaults.itSettings) *)
   .settings(
-    IntegrationTest / Keys.fork := false,
-    IntegrationTest / unmanagedSourceDirectories := (IntegrationTest / baseDirectory) (base => Seq(base / "it")).value,
+    IntegrationTest / Keys.fork                  := false,
+    IntegrationTest / unmanagedSourceDirectories := (IntegrationTest / baseDirectory)(base => Seq(base / "it")).value,
     addTestReportOption(IntegrationTest, "int-test-reports"),
     IntegrationTest / parallelExecution := false
   )
+
+//dependencyOverrides ++= Seq(
+//  "com.fasterxml.jackson.core"    % "jackson-databind"     % "2.14.3",
+//  "com.fasterxml.jackson.core"    % "jackson-core"         % "2.14.3",
+//  "com.fasterxml.jackson.core"    % "jackson-annotations"  % "2.14.3",
+//  "com.fasterxml.jackson.module" %% "jackson-module-scala" % "2.14.3"
+//)
```

Comment:
Could we remove this? Or could you add a description if it will be helpful in the future?

Reply:
Yes, thank you! Had been deciding between two options. Decided to upgrade with `"com.fasterxml.jackson.module" %% "jackson-module-scala"       % "2.15.2"` rather than downgrade with these commented ones to make them compatible with the scala version.